### PR TITLE
set height without margin to prevent agendaview to squeeze

### PR DESCRIPTION
**Pull request by amannepid**

### CODE REVIEW

**Comment by Tibolte** [comment]
It looks good to me, I'm currently on holidays, has anyone being able to test it?

File: agendacalendarview/src/main/java/com/github/tibolte/agendacalendarview/agenda/AgendaView.java [hunk 1]
```diff
@@ -67,7 +66,7 @@ public void onGlobalLayout() {
                                             ViewGroup.MarginLayoutParams layoutParams = (ViewGroup.MarginLayoutParams) getLayoutParams();
                                             int height = getHeight();
                                             int margin = (int) (getContext().getResources().getDimension(R.dimen.calendar_header_height) + 2 * getContext().getResources().getDimension(R.dimen.day_cell_height));
-                                            layoutParams.height = height - margin;
+                                            layoutParams.height = height;// - margin;
```
Comment (Tibolte) [inline agendacalendarview/src/main/java/com/github/tibolte/agendacalendarview/agenda/AgendaView.java]: @amannepid please remove that comment. let's not pollute the code with code that is not used anymore :)

**Comment by amannepid** [comment]
@Tibolte is it ok now ?

**Comment by Tibolte** [comment]
@amannepid yes :)

**Pull request merged**